src/components/StringFormatter/_index.scss

```diff
@@ -0,0 +1,20 @@
+////
+/// String formatter.
+/// @group string-formatter
+/// @copyright IBM Security 2019
+////
+
+@import '@carbon/layout/scss/mini-unit';
+
+@import '../Component/mixins';
+
+@include component($name: string-formatter) {
+  display: inline-block;
+
+  &--truncate {
+    max-width: carbon--mini-units($count: 84);
```

Comment:
I know I suggested changing this but now I'm questioning it. 😅 Since it comes out as `42rem` which is a bit different from what the original intent was, using the `ch` unit.